test/torch/nn/test_nn.py

```diff
@@ -4,7 +4,6 @@
 import torch.nn as nn
 import torch.nn.functional as F
 
-import syft.frameworks.torch.rnn as rnn
 import syft.frameworks.torch.nn as nn2
```

Comment:
Could you rename this to something else than `nn2`? `syft_nn` for instance, or just `nn` and `torch_nn` for the one above

Reply:
Done 😊